fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/api/TenantStatus.java

```diff
@@ -10,9 +10,9 @@
  * Tenant Status Enumeration
  */
 public enum TenantStatus {
-    PROVISIONING,
-    FREE,
-    ALLOCATED,
-    FROZEN,
-    DROPPED
+    PROVISIONING, // in the process of adding the partitions to the schema
+    FREE,         // unused tenant, available for allocation (e.g. tenant pooling)
```

Comment:
interesting idea of tenant pooling. that means we can  pre-allocate some tenants and then associate them later with tenant names.

Reply:
Yes, that was discussed to improve cloud provisioning time for new customers. Still a placeholder, because this requires additional work for cloud integration support.